openerp/addons/base/ir/ir_attachment.py

```diff
@@ -427,8 +430,26 @@ def create(self, cr, uid, values, context=None):
             values.pop(field, False)
         values = self._check_contents(cr, uid, values, context=context)
         self.check(cr, uid, [], mode='write', context=context, values=values)
+        if 'url' in values:
```

Comment:
But the trade-off is big (all asset cache is lost when you for example print an invoice). Do you see it really needed?